include/bluetooth/l2cap.h

```diff
@@ -21,7 +21,7 @@
 extern "C" {
 #endif
 
-#include <sys/atomic.h>
+#include <atomic.h>
```

Comment:
Is this a related change?

Reply:
It was unintended, it should be fixed now.